### ISSUE TITLE
Global LoadingBar component does not have isActive reactive prop.

### ISSUE THREAD

**Issue by StefanoMantero** [quasarframework/quasar#12059]
### What happened?

Currently, the global `loadingBar` component does not have a correctly implemented `isActive` reactive property. This has been asked [in this issue](https://github.com/quasarframework/quasar/issues/4071).

### What did you expect to happen?

In `loadingbar.vue` global file, the field `Status: active/idle` should update depending on the isActive prop that the global component have. This isn't happening as the Status is always in 'idle', since the `$q.loadingBar.isActive` is **undefined**.

### Reproduction URL

https://codepen.io/stefanomantero/pen/RwLdbXx

### How to reproduce?

1. Create fresh quasar dev environment.
2. Search for Loadingbar in GLOBAL
3. Go to Loadingbar
4. Add events both on $q or global
5. See the idle Status not change.

### Flavour

UMD

### Areas

Components (quasar), Plugins (quasar)

### Platforms/Browsers

Firefox, Chrome

### Quasar info output

```shell
Operating System        Linux(5.4.72-microsoft-standard-WSL2) - linux/x64
NodeJs                  16.3.0

Global packages
  NPM                   7.15.1
  yarn                  Not installed
  @quasar/cli           1.2.2
  @quasar/icongenie     Not installed
  cordova               Not installed

Networking
  Host                  DESKTOP-93Q6Q5H
  eth0                  172.22.108.122
```

**Comment by StefanoMantero** [comment]
I have created the following pull request #12062

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.4.11 and v1.17.6
